build.sbt

```diff
@@ -139,6 +146,7 @@ lazy val commonJvmSettings = Seq(
     val flag = if ((isTravisBuild in Global).value) "-oCI" else "-oDF"
     Tests.Argument(TestFrameworks.ScalaTest, flag)
   },
+  crossScalaVersions += "0.27.0-RC1",
```

Comment:
Maybe we should rename the `dotty_version` variable in the travis config to `scala_version_dotty`? WDYT?

Reply:
Yeah, sounds good. Or we shanghai @djspiewak into finishing #3570 😉